src/disjoint_set.jl

```diff
@@ -109,15 +143,15 @@ end
     intmap::Dict{T,Int}
     internal::IntDisjointSets
 
-    function (::Type{DisjointSets{T}}){T}(xs)    # xs must be iterable
+    function (::Type{DisjointSets{T}}){T}(xs, store_size::Bool=false)    # xs must be iterable
```

Comment:
I have added the second parameter for DisjointSet. Is there a way to write a constructor such that ```DisjointSets{Int}([1, 2, 3])``` still works?